http-clients/src/main/java/org/triplea/http/client/error/report/ErrorUploadClient.java

```diff
@@ -16,8 +14,8 @@
 @SuppressWarnings("InterfaceNeverImplemented")
 public interface ErrorUploadClient {
 
-  @VisibleForTesting
```

Comment:
No long visible for testing, the endpoint path definition re-uses this same constant. This way we guarantee that the endpoint referenced here is the same one we use for the http path.